tests/contrib/config.py

```diff
@@ -40,6 +40,14 @@
     'database': os.getenv('TEST_MYSQL_DATABASE', 'test'),
 }
 
+MARIADB_CONFIG = {
+    'host': '127.0.0.1',
+    'port': int(os.getenv('TEST_MYSQL_PORT', 3306)),
```

Comment:
Should we use mariadb specific variables? `TEST_MARIADB_PORT`, etc?

Reply:
sounds good!